### ISSUE TITLE
Filtered deck created on ankidroid isn't studyable on anki desktop

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#2585]
Originally reported on Google Code with ID 1683

```
What steps will reproduce the problem?
1.  Made subdeck on ankidroid (2.0 beta?) (from deck originally created on desktop
anki (1._)
2.  Opened subdeck on laptop
3.  When try to answer first card, get error message.  Cannot answer any cards
4.  Can answer cards on ankidroid

What is the expected output? What do you see instead?

error message:
Traceback (most recent call last):
  File "aqt/webview.pyc", line 21, in link
  File "aqt/reviewer.pyc", line 289, in _linkHandler
  File "aqt/reviewer.pyc", line 227, in _showAnswer
  File "aqt/reviewer.pyc", line 570, in _showEaseButtons
  File "aqt/reviewer.pyc", line 619, in _answerButtons
  File "aqt/reviewer.pyc", line 613, in but
  File "aqt/reviewer.pyc", line 628, in _buttonTime
  File "anki/sched.pyc", line 1133, in nextIvlStr
  File "anki/utils.pyc", line 58, in fmtTimeSpan
  File "anki/utils.pyc", line 73, in optimalPeriod
TypeError: bad operand type for abs(): 'unicode'

Does it happen again every time you repeat the steps above? Or did it
happen only one time?
every time

What version of AnkiDroid are you using? (Decks list > menu > About > Look
at the title)
desktop: 2.0-rc4
droid:  Ankidroid v2.0, but i JUST updated from the beta today (after I created the
subdeck)

On what version of Android? (Home screen > menu > About phone > Android
version)
Jellybean (4.1.1)

If it is a crash or "Force close" and you can reproduce it, the following
would help immensely: 1) Install the "SendLog" app, 2) Reproduce the crash,
3) Immediately after, launch SendLog, 4) Attach the resulting file to this
report. That will make the bug much easier to fix.
basically crashes the desktop client.  First time froze it, next time just couldn't
use that deck.

Please provide any additional information below.

I already posted this on the anki desktop bug forum, but they booted me over here,
saying it's a droid problem.
```

Reported by `joshua.frenkel` on 2013-02-13 02:03:38
